### PR TITLE
Add resource page

### PR DESCRIPTION
**Pull request by c0demat1cs**
Deleted the Expenses and Activities page. 
Created a Resource page and updated it with appropriate content/styling

### CODE REVIEW

**Review by grangjer — COMMENTED** [review]
Hey guys. I have reviewed and all looks great to me. I have no changes to request at the moment, but I won't approve as of now just in case everyone else wants to take a look at it beforehand.

Thanks.

**Review by YoungwooWatson — APPROVED** [review]
As I mentioned on Discord, I’ve already reviewed the code and I think it is great! Since Isadora has also given her approval, and we now have two people in agreement, I have approve the ‘pull request’. You will now have access to merge the code.

Thank you for the work you’ve put into this!

**Pull request merged**
